readme.md

```diff
@@ -1,5 +1,39 @@
-# Ojvindix
+### Ojvindix
 
-Homemade Cémantix
+This project is a simple word game built with React and nodejs. The game allows multiple players to connect and submit words. A similarity score is given based on the similarity of the word to the target word. The player wins when he finds the target word
 
-https://www.ojvindix.fr/
\ No newline at end of file
+## Requirements
+
+-    Node.js
+-    React
+-    npm
+
+## Installation
+
+Clone the repository:
+
+    git clone https://github.com/user/repo.git](https://github.com/HugoOjvindFrancois/ojvindix
+
+Install dependencies:
+
+    npm install
+
+Start the development server:
+
+    npm start
+
+## Usage
+  #### Singleplayer
+  
+  To submit a word, enter it in the input field and click the "Submit" button. The similarity score is based on the similarity of the word to the target word. When you find the target word you win the game
+
+  #### Multiplayer
+  To connect to the game, enter a multiplayer code and a username, then click the "Connect" button. The game will automatically receive and display words submitted by other players.
+
+## Features 
+  A DOOM mode is available when you press the DOOM button on the bottom left of the screen
+  A timer is on the bottom right of the screen, you can see how fast you find the word of the day
+
+## License
+
+This project is licensed under the terms of the MIT license.
```

Comment:
The project is protected with the [GPL-3.0 license](https://github.com/HugoOjvindFrancois/ojvindix/blob/develop/LICENSE.md) not the MIT one